app/helpers/table_helper.rb

```diff
@@ -0,0 +1,42 @@
+module TableHelper
+  def sort_table_header(heading, content_item_attribute, order, sort)
+    # Determine if this column is currently sorted on
+    # Generate labels and aria attributes accordingly
+    # Generate link to sort by this column asc || desc
+    currently_sorted = order.present? && sort == content_item_attribute
+    next_sort = next_sort(currently_sorted, heading, order)
+    current_sort = current_sort(currently_sorted, heading, order)
+    content_tag :th, "aria-sort" => current_sort[:aria_attr] do
+      sort_link heading, next_sort[:label], content_item_attribute, next_sort[:param]
+    end
+  end
+
+private
+
+  def sort_link(heading, description, content_item_attribute, order)
+    link_content = heading + " " + (content_tag :span, description, class: "visually-hidden")
+    link_content = link_content.html_safe
+    link_to link_content, organisation_content_items_path(
```

Comment:
You can use a [block in link_to](http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to) to define the link content.